### ISSUE TITLE
v2.0  Joining procedure via QR Code - Suggestion for scan process

### ISSUE THREAD

**Issue by lestaticon** [ThomasGaubert/zephyr#59]
Android app version:
2.0.0-beta-b45d554

Joining procedure via QR Code

Steam app:
I click QR Code and the code and IP are displayed. 

Android app:
Even though the doc says to press and hold the blue Start button, the most natural direction for me was to click "No join code saved" from the main screen. I was glad to see the choice slide in appear. I chose Scan QR code. I'm asked to give permission to take pictures and record video. After accepting, I then see the video output and align my camera with the code on my Steam app. There is no feedback or indication where to position the camera. Only after some trial and error did it complete. I only suggest to add some haptic feedback or sound for when it is successful and some way to guide the user to the correct camera position. 

About device:
Android version 9
Moto Z3 Play

**Comment by ThomasGaubert** [comment]
Thanks for the feedback! This is something I intend to address prior to release.

**Comment by ThomasGaubert** [comment]
Added vibration effect in 73922558ed2b4c3ae3362853ebe87ac9ab6b90ac (will be included in future beta release). I tried adding an overlay to show what QR codes the vision API sees, but it's not useful unless you're scanning non-Zephyr codes. 

The vision API should scan any visible QR code so a target overlay isn't necessary at this point. I'll add some additional help content and toasts when scanning takes longer than expected.
